crates/bevy_text/src/pipeline.rs

```diff
@@ -14,28 +13,19 @@ use crate::{
     TextAlignment, TextSection,
 };
 
-pub struct TextPipeline<ID> {
+#[derive(Default)]
+pub struct TextPipeline {
     brush: GlyphBrush,
-    glyph_map: HashMap<ID, TextLayoutInfo>,
     map_font_id: HashMap<HandleId, FontId>,
 }
 
-impl<ID> Default for TextPipeline<ID> {
-    fn default() -> Self {
-        TextPipeline {
-            brush: GlyphBrush::default(),
-            glyph_map: Default::default(),
-            map_font_id: Default::default(),
-        }
-    }
-}
-
+#[derive(Component, Clone, Default, Debug)]
 pub struct TextLayoutInfo {
```

Comment:
This will be much more visible; can you add some docs?

Reply:
Yes!